### ISSUE TITLE
cljfx reflection warning on un-resolved method updateItem on javafx.scene.control.TableCell

### ISSUE THREAD

**Issue by stardiviner** [cljfx/cljfx#5]
I run `re-find.fx` which using `cljfx` as GUI library.
Here is the command:
```
clj -Sdeps "{:deps                   {MageMasher/re-find.fx                      {:git/url \"https://github.com/MageMasher/re-find.fx\"                       :sha \"b29bde3519f7632b63eb71415943bf8a7cfa1462\"}}}" \    -m re-find.fx
```
Then I got following wanring.
```
Reflection warning, cljfx/fx/table_cell.clj:33:11 - call to method updateItem on javafx.scene.control.TableCell can't be resolved (no such method).
```

**Comment by vlaaad** [comment]
Fixed in 551c5e1dc8c248c7ed27d72bc02e5a40116712c9, released as [1.2.2](https://github.com/cljfx/cljfx/blob/master/CHANGELOG.md#122---2019-03-19)

**Comment by stardiviner** [comment]
@vlaaad Thanks!
